models.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import Column, Integer, String, func, DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBAuthor(Base):
+    __tablename__ = "author"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(127), unique=True, nullable=False)
+    bio = Column(String(511), nullable=True)
+
```

Comment:
You did not include field for relation with book, and this will not cause some errors or whatever. BUT
You will not have backward relation and when you load the author, you will not be able to `author.books`. To retrieve all books written by some author - you'll have to add additional query by yourself.

And also, task requirement asked you to do that:)